### PR TITLE
Security Guide Formatting fixes

### PR DESCRIPTION
**Pull request by sheet20**
## What Changed?

While working on v11 migration for Security Features Guide came across formatting issues, fixed.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/epas/12/epas_security_guide/02_protecting_against_sql_injection_attacks/01_sql_protect_overview.mdx [hunk 1]
```diff
@@ -12,15 +12,15 @@ This section contains an introduction to the different types of SQL injection at
 
 ## Types of SQL Injection Attacks
 
-There are a number of different techniques used to perpetrate SQL injection attacks. Each technique is characterized by a certain `signature`. SQL/Protect examines queries for the following signatures:
+There are a number of different techniques used to perpetrate SQL injection attacks. Each technique is characterized by a certain *signature*. SQL/Protect examines queries for the following signatures:
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_security_guide/02_protecting_against_sql_injection_attacks/01_sql_protect_overview.mdx:15]: retain back ticks

File: product_docs/docs/epas/12/epas_security_guide/02_protecting_against_sql_injection_attacks/01_sql_protect_overview.mdx [hunk 2]
```diff
@@ -12,15 +12,15 @@ This section contains an introduction to the different types of SQL injection at
...
 
 **Unauthorized Relations**
 
-While Advanced Server allows administrators to restrict access to relations (tables, views, etc.), many administrators do not perform this tedious task. SQL/Protect provides a `learn` mode that tracks the relations a user accesses.
+While Advanced Server allows administrators to restrict access to relations (tables, views, etc.), many administrators do not perform this tedious task. SQL/Protect provides a *learn* mode that tracks the relations a user accesses.
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_security_guide/02_protecting_against_sql_injection_attacks/01_sql_protect_overview.mdx:19]: retain back ticks

File: product_docs/docs/epas/12/epas_security_guide/02_protecting_against_sql_injection_attacks/01_sql_protect_overview.mdx [hunk 3]
```diff
@@ -44,32 +44,32 @@ A dangerous action taken during SQL injection attacks is the running of unbounde
...
--   The statistic in column superusers of `edb_sql_protect_stats` is incremented with every command issued by the protected superuser. See `Attack Attempt Statistics` for information on the `edb_sql_protect_stats` view.
+-   The statistic in column superusers of `edb_sql_protect_stats` is incremented with every command issued by the protected superuser. See *Attack Attempt Statistics* for information on the `edb_sql_protect_stats` view.
 -   When SQL/Protect is in active mode, all commands issued by the protected superuser are prevented from running.
 
 A protected role that has the superuser privilege should either be altered so that it is no longer a superuser, or it should be reverted back to an unprotected role.
 
 ### Attack Attempt Statistics
 
-Each usage of a command by a protected role that is considered an attack by SQL/Protect is recorded. Statistics are collected by type of SQL injection attack as discussed in `Types of SQL Injection Attacks`.
+Each usage of a command by a protected role that is considered an attack by SQL/Protect is recorded. Statistics are collected by type of SQL injection attack as discussed in *Types of SQL Injection Attacks*.
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_security_guide/02_protecting_against_sql_injection_attacks/01_sql_protect_overview.mdx:65]: this should be a link.

**Review by drothery-edb — CHANGES_REQUESTED** [review]
please review all your changes and retain the back ticks for all keywords. They shouldn't be italics. Also, any references to other sections should be links.

**Comment by sheet20** [comment]
Will incorporate these changes later, closing the request as of now.

**Pull request closed without merging**